KIPs/kip-7-fungible_token.md

```diff
@@ -47,9 +49,38 @@ If the optional field is not marked, the function must be implemented.
 |[name](#name)|O|function name() public view returns (string)|
 |[symbol](#symbol)|O|function symbol() public view returns (string)|
 |[decimals](#decimals)|O|function decimals() public view returns (uint8)|
-|[mint](#mint)| O |function mint(address _to, uint256 _value) public returns (bool) |
-|[burn](#burn)| O |function burn(uint256 __value) public |
-|[burnFrom](#burnfrom)| O |function burnFrom(address _from, uint256 _value) public |
+
+#### Mint Methods (Optional)
+The table below is a summary of methods related with Mint function.
+These methods are optional.
+
+|Name|Prototype|
+|---|---|
+|[mint](#mint)|function mint(address _to, uint256 _value) public onlyMinter returns (bool) |
+|[isMinter](#isminter)|function isMinter(address _account) public view returns (bool) |
+|[addMinter](#addminter)|function addMinter(address _account) public onlyMinter |
+|[renounceMinter](#renounceminter)|function renounceMinter() public |
+
+#### Burn Methods (Optional)
+The table below is a summary of methods related with Burn function.
+These methods are optional.
+
+|Name|Prototype|
+|---|---|
+|[burn](#burn)|function burn(uint256 _value) public |
+|[burnFrom](#burnfrom)|function burnFrom(address _from, uint256 _value) public |
+
+#### Pause Methods (Optional)
```

Comment:
Pause also has 'addPauser' and 'renouncePauser' functions.
Can you add these functions too?

Reply:
Yes I will~! :)